README.md

```diff
@@ -79,18 +104,31 @@ On KnossosNET there are multiples sources of Command Line arguments that are eve
 
 <br /><br />
 ## **Current NuGet Packages:**<br />
-- Avalonia 11.0.5<br />
-- Avalonia.Desktop 11.0.5<br />
-- Avalonia.Diagnostics 11.0.5<br />
-- Avalonia.Themes.Fluent 11.0.5<br />
+- Avalonia 11.2.3<br />
+- Avalonia.Desktop 11.2.3<br />
+- Avalonia.Control.DataGrid 11.2.3<br />
+- Avalonia.Diagnostics 11.2.3<br />
+- Avalonia.Markup.Xaml.Loader 11.2.3<br />
+- Avalonia.Themes.Fluent 11.2.3<br />
 - Avalonia.HtmlRenderer 11.0.0<br />
-- CommunityToolkit.Mvvm 8.2.1<br />
+- CommunityToolkit.Mvvm 8.4.0<br />
+- Microsoft.Extensions.Http 8.0.1<br />
 - ini-parser-netstandard 2.5.2<br />
-- SharpCompress 0.33.0<br />
+- SharpCompress 0.39.0<br />
 - AnimatedImage.Avalonia 1.0.7<br />
+- ObservableCollections 3.3.3<br />
+- WindowsShortcutFactory 1.2.0<br />
 
 <br /><br />
 ## **Compiling for Linux and Mac:**<br />
 - Right click on the project -> Publish<br />
 - Export to folder<br />
-- From there you can pick the dest enviroment on the list<br />
+- From there you can pick the destiny environment on the list<br />
+<br /><br />
+
+# Custom TC Launcher Mode<br />
+![CustomLauncher](https://i.imgur.com/uCYnTvD.png)
+<br />
+<br />
+Starting from 1.3.0 Knossos.NET supports "Custom TC Mode" is can be used to provide a customized launcher for a specific Total Conversion mod. Supporting customization by an additional json file or by hard-coding the data in.<br />
```

Comment:
Suggested change
`Starting from 1.3.0, Knossos.NET supports "Custom TC Mode" which can be used to provide a customized launcher for a specific Total Conversion mod. "Custom TC Mode" supports customization via an additional json file or by hard-coding the data in.`